### DISCUSSION TITLE
Improve nginx config for Lavavel (Glide not working in Statamic CMS Quickstart)

### DISCUSSION THREAD

**Discussion by st1pps** [ddev/ddev#6751] [General]
I just tried to get Statamic CMS working in DDEV, and followed the guide here: https://ddev.readthedocs.io/en/stable/users/quickstart/#statamic

The page itself works, but when I tried to resize images in my templates with Glide, no picture would be displayed. Turns out that the following part of the nginx configuration strips the last query parameter, which causes it to fail: 

```
# Media: images, icons, video, audio, HTC
    location ~* \.(?:jpg|jpeg|gif|png|ico|cur|gz|svg|svgz|mp4|ogg|ogv|webm|webp|htc)$ {
        expires 1M;
        access_log off;
        add_header Cache-Control "public";
    }
```

See https://github.com/statamic/cms/discussions/11146#discussioncomment-11324539
Would it make sense to add that information to the CMS quickstart for Statamic, or change the nginx conf to something that would work for Statamic as well?

**Comment by rfay** [comment]
Hi @st1pps - 

Since Statamic doesn't have an explicit project type, it's hard-ish to script-automate a simple addition to Nginx snippets or nginx_full. 

Could you push up a demo repository that demonstrates this?

It's worth considering that in the DDEV quickstart, Statamic is just a `laravel` project, so it would be easy to add that to https://github.com/ddev/ddev/blob/master/pkg/ddevapp/webserver_config_assets/nginx-site-laravel.conf if it does no harm. You could do a PR for that.

I'm sure you know that you can fix this in your project with config in .ddev/nginx or .ddev/nginx_full.

**Reply by rfay** [reply]
I created an issue from this, I think we can just improve the laravel default nginx config
* https://github.com/ddev/ddev/issues/6774

**Comment by tyler36** [comment]
Created https://github.com/ddev/ddev/pull/6775 to resolve issue.

**Reply by rfay** [reply]
@st1pps please test the artifacts in https://github.com/ddev/ddev/pull/6775#issuecomment-2499381426 and see if they resolve your issue, and comment/review there, thanks.

**Reply by st1pps** [reply]
@rfay I can confirm that with that build my Statamic page runs as intended :) 
The nginx config was newly generated and works properly with glide. 
Thanks a lot!
